lib/IRGen/GenProto.cpp

```diff
@@ -2528,6 +2532,16 @@ void IRGenModule::emitSILWitnessTable(SILWitnessTable *wt) {
   bool isResilient = isResilientConformance(conf);
   bool useRelativeProtocolWitnessTable =
     IRGen.Opts.UseRelativeProtocolWitnessTables;
+  if (useRelativeProtocolWitnessTable &&
```

Comment:
@slavapestov I am trying to test for variadic generics with conditional conformances here. Probably, doing it all wrong?

Reply:
Well, for one a `dump()` isn't going to fly in an no-asserts build ...

Reply:
It looks fine :)